desktop/appcontainer/Message.js

```diff
@@ -53,32 +55,37 @@ export class Message extends Component {
     }
 
     renderInput() {
-        const {formModel, input} = this.model;
+        const {formModel, input, autoFocus} = this.model;
         if (!formModel) return null;
         return form({
             model: formModel,
             fieldDefaults: {commitOnChange: true, minimal: true, label: null},
             item: formField({
                 field: 'value',
-                item: withDefault(input.item, textInput({autoFocus: true}))
+                item: withDefault(input.item, textInput({
+                    autoFocus:  autoFocus == 'input' ? true : false
```

Comment:
Why the ternary operator? Could simplify to just `autoFocus == 'input'` - same below in `renderButtons()`.

Here its a bit tricky as any custom input (e.g. a numberInput) won't get focused. That puts me on the fence with supporting input as an option, although I can certainly see the use.  We could clone the element if provided and install as an override - that might be reasonable here.